src/model/auditRecords.coffee

```diff
@@ -0,0 +1,56 @@
+mongoose = require "mongoose"
+Schema = mongoose.Schema
+
+eventIdSchema = new Schema
+  "code": { type: String, required: false }
+  "codeSystemName": { type: String, required: false }
+  "displayName": { type: String, required: false }
```

Comment:
Not a big deal, but we could probably use a single schema for all the types that use code, codeSystemName and displayName. A good name would be `CodedType` or something similar.